include/lsst/afw/cameraGeom/Detector.h

```diff
@@ -150,6 +150,28 @@ class Detector {
     lsst::afw::table::AmpInfoRecord const & operator[](std::string const &name) const;
 
     /**
+     * Get the amplifier specified by index as a shared_ptr
+     *
+     * @warning Intended only for internal use by pybind11
```

Comment:
I added this comment: "This exists because Operator[] returns a data type that is harder for pybind11 to use."